Java/CheckPositiveOrNegative.java

```diff
@@ -0,0 +1,23 @@
+public class CheckPositiveOrNegative  
+{  
+public static void main(String[] args)   
+{  
+//number to be check  
+int num=912;  
+//checks the number is greater than 0 or not  
+if(num>0)  
+{  
+System.out.println("The number is positive.");  
+}  
+//checks the number is less than 0 or not  
+else if(num<0)  
+{  
+System.out.println("The number is negative.");  
+}  
+//executes when the above two conditions return false  
+else  
+{  
+System.out.println("The number is zero.");  
+}  
+}  
+}  
```

Comment:
Are you writing C or JAVA?

If you're writing JAVA code please indent properly and follow best practices.

```java
public class CheckPositiveOrNegative {
    public static void main(String[] args) {
        //number to be check  
        int num = 912;
        //checks the number is greater than 0 or not  
        if (num > 0) {
            System.out.println("The number is positive.");
        }
        //checks the number is less than 0 or not  
        else if (num < 0) {
            System.out.println("The number is negative.");
        }
        //executes when the above two conditions return false  
        else {
            System.out.println("The number is zero.");
        }
    }
}
```